src/lgtm.js

```diff
@@ -50,15 +50,15 @@ function submitAsReview(message) {
         if (!expandedAttr || expandedAttr.nodeValue == 'false') reviewToggle.click();
 
         // select the 'approve' state
-        var approve = document.querySelector("#submit-review[aria-expanded=true] input[value=approve]");
+        var approve = document.querySelector('input[name="pull_request_review[event]"][value="approve"]');
         if (approve) approve.click();
 
         // set lgtm
         var input = document.getElementById("pull_request_review_body");
         handleMessage(input, message);
 
         if (message.type != lgtmNoSubmitType) {
-            var submitBtn = document.querySelector("#submit-review[aria-expanded=true] [type=submit]");
+            var submitBtn = document.querySelector("div.form-actions button[type=submit]");
```

Comment:
you don't need the `div.form-actions`, just `.form-actions` should be enough and more flexible